terraform-random-pet/README.md

```diff
@@ -0,0 +1,102 @@
+# terraform-random-pet
+
+This is a template child module.
+
+## Documentation Recommendations (DO NOT INCLUDE THIS INTO THE REAL README)
+
+### Naming
+
+The repository/directory name should follow this pattern:
+
+```sh
+terraform-<PROVIDER>-<NAME>
+```
+
+Here’s what this means:
+
+1. The repository should start with `terraform-` if your module should be [published to and discovered on the Registry](https://opentofu.org/docs/language/modules/develop/publish/). Even if you don’t intend to publish the module, following this pattern is a good practice that helps differentiate your Terraform child modules from other code in your projects.
```

Comment:
I think we should only recommend the `terraform-` prefix IF it's a separate repo. If it's a child-module that is in the directory structure, then that feels a bit overkill, right?

Reply:
Clarified.